app/serializers/pizza_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class PizzaSerializer < ActiveModel::Serializer
+  attributes :id, :product_name, :cost
+
+  belongs_to :brand, serializer: BrandSerializer
```

Comment:
I'm curious why you added BrandSerializer here? You should be able to just leave that to the controller.

Reply:
We added the BrandsSerializer here so that we would only get the attributes from brand that we wanted, not all of them from the controller (like `created_at:` and `updated_at:`).

Reply:
So it's the controller that would be calling on the serializer. You might want to take a peek back at Nick's clinic he gave with the serializers. We want to try to keep that localized to the controller itself. Nick's clinic had a method with each (each serializer?). I think that will give you what you're trying to do.